src/main/c/cext/numeric.c

```diff
@@ -161,6 +161,10 @@ VALUE rb_ull2inum(unsigned LONG_LONG val) {
   return rb_tr_wrap(polyglot_invoke(RUBY_CEXT, "rb_ulong2num", val));
 }
 
+int rb_uv_to_utf8(char* buf, unsigned long uv) {
+  return polyglot_as_i64(polyglot_invoke(RUBY_CEXT, "rb_uv_to_utf8", buf, uv));
+}
```

Comment:
This would be better in `encoding.c`, since it's a conversion between a Unicode codepoint to a char*.